tests/h/views/accounts_test.py

```diff
@@ -255,7 +255,6 @@ def test_post_creates_no_activations_when_validation_fails(
     def test_post_generates_mail(
```

Comment:
Holy crap. This test is 933 lines long! Fair enough the file it's testing is 633 so the ratio isn't bad, but this could do with splitting up a bit!